unit_tests/util/packed_vector.cpp

```diff
@@ -205,73 +202,4 @@ BOOST_AUTO_TEST_CASE(packed_vector_33bit_continious)
     }
 }
 
-BOOST_AUTO_TEST_CASE(packed_weights_container_with_type_erasure)
```

Comment:
`std::ranges::*` don't have anything like `boost::any_range` and we actually don't use anything like this, so this test seems to have no value